synapseutils/sync.py

```diff
@@ -614,8 +614,8 @@ def _upload_item(
 
 def generateManifest(syn, allFiles, filename, provenance_cache=None):
     """Generates a manifest file based on a list of entities objects.
-
-    :param allFiles:   A list of File Entities
+    :param syn:   A synapse object as obtained with syn = synapseclient.login()
+    :param allFiles:   A list of File Entities on Synapse 
```

Comment:
```suggestion
    :param allFiles:   A list of File Entities on Synapse
```

Reply:
Thanks Dan,  does this have to be a list of `File` entity objects, or can it be synapse ids?